doofinder-for-woocommerce/includes/settings/class-renderers.php

```diff
@@ -96,6 +97,12 @@ private function render_html_settings()
             $store_api->normalize_store_and_indices();
         }
 
+        if (isset($_GET['clear_landing_cache'])) {
+            echo "<h1>Conversion page caché is clear</h1>";
```

Comment:
In English, the word "cache" should not contain any accent.

Reply:
ups